README.md

```diff
@@ -164,7 +164,7 @@ $ swiftlint help
 Available commands:
 
    analyze         [Experimental] Run analysis rules
-   autocorrect     Automatically correct warnings and errors
+   fix           Automatically correct warnings and errors
```

Comment:
This is incorrect. Can you run `swiftlint help` and paste the response here?

Reply:
Did it

Reply:
I figured out that the `fix` not appear in help, is there any chance to create a issue and/or PR adding?